packages/hardhat/deploy/00_deploy_your_contract.ts

```diff
@@ -25,28 +25,39 @@ const deployYourContract: DeployFunction = async function (hre: HardhatRuntimeEn
   const { deployer } = await hre.getNamedAccounts();
   const { deploy } = hre.deployments;
 
-  await deploy("BatchRegistry", {
+  // await deploy("BatchRegistry", {
+  //   from: deployer,
+  //   // Contract constructor arguments
+  //   args: [deployer, BATCH_NUMBER],
+  await deploy("PixelCanvas", {
     from: deployer,
-    // Contract constructor arguments
-    args: [deployer, BATCH_NUMBER],
+    args: [],
     log: true,
     // autoMine: can be passed to the deploy function to make the deployment process faster on local networks by
     // automatically mining the contract deployment transaction. There is no effect on live networks.
     autoMine: true,
   });
 
   // Get the deployed contract to interact with it after deploying.
-  const batchRegistry = await hre.ethers.getContract<Contract>("BatchRegistry", deployer);
-  console.log("\nBatchRegistry deployed to:", await batchRegistry.getAddress());
-  console.log("Remember to update the allow list!\n");
+  // const batchRegistry = await hre.ethers.getContract<Contract>("BatchRegistry", deployer);
+  // console.log("\nBatchRegistry deployed to:", await batchRegistry.getAddress());
+  // console.log("Remember to update the allow list!\n");
```

Comment:
we still need to deploy the BatchRegistry, right?

Reply:
I actually don't have idea about it that's why i commented it out till @phipsae replied

Reply:
As a reviewer of the PR, I would first test it in my local environment, which involves deploying all the contracts (including batchRegistry). Therefore, you should add the deployment of your new contract to the existing script (so you leave the script as it is and add the additional deploy code).